pkg/minikube/cluster/cluster.go

```diff
@@ -231,18 +242,20 @@ func trySSHPowerOff(h *host.Host) {
 }
 
 // StopHost stops the host VM, saving state to disk.
-func StopHost(api libmachine.API) error {
+func StopHost(api libmachine.API, config cfg.MachineConfig) error {
 	host, err := api.Load(cfg.GetMachineName())
 	if err != nil {
 		return errors.Wrapf(err, "load")
 	}
-	console.OutStyle(console.Stopping, "Stopping %q in %s ...", cfg.GetMachineName(), host.DriverName)
-	if err := host.Stop(); err != nil {
-		alreadyInStateError, ok := err.(mcnerror.ErrHostAlreadyInState)
-		if ok && alreadyInStateError.State == state.Stopped {
-			return nil
+	if config.VMDriver != constants.DriverNone {
+		console.OutStyle(console.Stopping, "Stopping %q in %s ...", cfg.GetMachineName(), host.DriverName)
+		if err := host.Stop(); err != nil {
+			alreadyInStateError, ok := err.(mcnerror.ErrHostAlreadyInState)
+			if ok && alreadyInStateError.State == state.Stopped {
+				return nil
+			}
+			return &util.RetriableError{Err: errors.Wrapf(err, "Stop: %s", cfg.GetMachineName())}
 		}
```

Comment:
If "stop" is unsupported by none, I think StopHost should return an unsupported error instead of nothing.

Reply:
Well, StopHost is unsupported - but we should try to stop **Kubernetes**. Just TBD.

Maybe I should add a stopKubernetes() function, similar to our uninstallKubernetes()

And just make it empty or log something friendly...

The problem is described in that other issue linked.